libraries/app/database_api.cpp

```diff
@@ -2009,9 +2009,10 @@ set<public_key_type> database_api_impl::get_potential_signatures( const signed_t
             result.insert(k);
          return &auth;
       },
+      &result,
       _db.get_global_properties().parameters.max_authority_depth
    );
-
+   /*
```

Comment:
Please remove this code instead of only commenting it out.

Reply:
ok，already removed.